### PR TITLE
Update dependencies

### LINKED ISSUE #309: Dependencies update
It's that time again folks! deps upgrade for f/e 🎉
Only just over a month since #52 and look at it!

`yarn outdated --json | $(yarn bin)/format-yarn-outdated --format markdown`

# Major
| Package | Current | Wanted | Latest | Package Type | URL | CHANGELOG |
|---|---|---|---|---|---|---|
| @storybook/testing-library | 0.0.11 | 0.0.11 | 0.0.13 | openmsupply-client | devDependencies | https://github.com/storybookjs/testing-library#readme |  |
| @types/jest | 27.5.1 | 27.5.2 | 28.1.4 | openmsupply-client | devDependencies | https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/jest |  |
| @types/node | 17.0.35 | 17.0.45 | 18.0.3 | openmsupply-client | devDependencies | https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/node |  |
| copy-webpack-plugin | 10.2.4 | 10.2.4 | 11.0.0 | @openmsupply-client/electron | devDependencies | https://github.com/webpack-contrib/copy-webpack-plugin |  |
| electron | 18.2.4 | 18.2.4 | 19.0.8 | @openmsupply-client/electron | devDependencies | https://github.com/electron/electron#readme |  |
| fork-ts-checker-webpack-plugin | 6.5.2 | 6.5.2 | 7.2.11 | @openmsupply-client/electron | devDependencies | https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/common | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/config | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/dashboard | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/host | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/inventory | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/invoices | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/requisitions | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| lint-staged | 12.4.2 | 12.5.0 | 13.0.3 | @openmsupply-client/system | devDependencies | https://github.com/okonet/lint-staged#readme |  |
| msw | 0.40.2 | 0.40.2 | 0.43.1 | openmsupply-client | dependencies | https://mswjs.io/ |  |
| react-refresh | 0.13.0 | 0.13.0 | 0.14.0 | @openmsupply-client/host | devDependencies | https://reactjs.org/ |  |
| zustand | 3.7.2 | 3.7.2 | 4.0.0-rc.1 | @openmsupply-client/common | dependencies | https://github.com/pmndrs/zustand |  |

# Minor
| Package | Current | Wanted | Latest | Package Type | URL | CHANGELOG |
|---|---|---|---|---|---|---|
| @awesome-cordova-plugins/core | 5.43.0 | 5.44.0 | 5.44.0 | @openmsupply-client/system | devDependencies | https://github.com/danielsogl/awesome-cordova-plugins#readme |  |
| @awesome-cordova-plugins/printer | 5.43.0 | 5.44.0 | 5.44.0 | @openmsupply-client/system | devDependencies | https://github.com/danielsogl/awesome-cordova-plugins#readme |  |
| @bugsnag/js | 7.16.5 | 7.17.0 | 7.17.0 | openmsupply-client | dependencies | https://www.bugsnag.com/ |  |
| @bugsnag/js | 7.16.5 | 7.17.0 | 7.17.0 | @openmsupply-client/common | dependencies | https://www.bugsnag.com/ |  |
| @capacitor/android | 3.5.1 | 3.6.0 | 3.6.0 | @openmsupply-client/android | devDependencies | https://capacitorjs.com |  |
| @capacitor/cli | 3.5.1 | 3.6.0 | 3.6.0 | @openmsupply-client/android | devDependencies | https://capacitorjs.com |  |
| @capacitor/core | 3.5.1 | 3.6.0 | 3.6.0 | @openmsupply-client/android | devDependencies | https://capacitorjs.com |  |
| @capacitor/core | 3.5.1 | 3.6.0 | 3.6.0 | @openmsupply-client/common | dependencies | https://capacitorjs.com |  |
| @emotion/styled | 11.8.1 | 11.9.3 | 11.9.3 | @openmsupply-client/common | dependencies | https://github.com/emotion-js/emotion/tree/main/packages/styled |  |
| @graphql-codegen/cli | 2.6.2 | 2.7.0 | 2.7.0 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @graphql-codegen/typescript | 2.4.11 | 2.6.0 | 2.6.0 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @testing-library/dom | 8.13.0 | 8.14.0 | 8.14.0 | openmsupply-client | devDependencies | https://github.com/testing-library/dom-testing-library#readme |  |
| @testing-library/react | 13.2.0 | 13.3.0 | 13.3.0 | openmsupply-client | devDependencies | https://github.com/testing-library/react-testing-library#readme |  |
| @typescript-eslint/eslint-plugin | 5.26.0 | 5.30.5 | 5.30.5 | openmsupply-client | devDependencies | https://github.com/typescript-eslint/typescript-eslint#readme |  |
| @typescript-eslint/eslint-plugin | 5.26.0 | 5.30.5 | 5.30.5 | @openmsupply-client/electron | devDependencies | https://github.com/typescript-eslint/typescript-eslint#readme |  |
| @typescript-eslint/parser | 5.26.0 | 5.30.5 | 5.30.5 | openmsupply-client | devDependencies | https://github.com/typescript-eslint/typescript-eslint#readme |  |
| @typescript-eslint/parser | 5.26.0 | 5.30.5 | 5.30.5 | @openmsupply-client/electron | devDependencies | https://github.com/typescript-eslint/typescript-eslint#readme |  |
| eslint | 8.16.0 | 8.19.0 | 8.19.0 | openmsupply-client | devDependencies | https://eslint.org |  |
| eslint | 8.16.0 | 8.19.0 | 8.19.0 | @openmsupply-client/electron | devDependencies | https://eslint.org |  |
| eslint-plugin-storybook | 0.5.12 | 0.5.13 | 0.5.13 | openmsupply-client | devDependencies | https://github.com/storybookjs/eslint-plugin-storybook#readme |  |
| graphql-request | 4.2.0 | 4.3.0 | 4.3.0 | @openmsupply-client/common | dependencies | https://github.com/prisma/graphql-request |  |
| lerna | 5.0.0 | 5.1.8 | 5.1.8 | openmsupply-client | dependencies | https://lerna.js.org |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/common | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/config | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/dashboard | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/host | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/inventory | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/invoices | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/requisitions | devDependencies | https://prettier.io |  |
| prettier | 2.6.2 | 2.7.1 | 2.7.1 | @openmsupply-client/system | devDependencies | https://prettier.io |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | openmsupply-client | devDependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/common | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/dashboard | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/electron | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/host | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/inventory | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/invoices | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/requisitions | dependencies | https://reactjs.org/ |  |
| react | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/system | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | openmsupply-client | devDependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/common | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/dashboard | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/electron | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/host | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/inventory | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/invoices | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/requisitions | dependencies | https://reactjs.org/ |  |
| react-dom | 18.1.0 | 18.2.0 | 18.2.0 | @openmsupply-client/system | dependencies | https://reactjs.org/ |  |
| react-i18next | 11.16.9 | 11.18.0 | 11.18.0 | @openmsupply-client/common | dependencies | https://github.com/i18next/react-i18next |  |
| swc-loader | 0.2.1 | 0.2.3 | 0.2.3 | @openmsupply-client/host | dependencies | https://github.com/swc-project/swc-loader |  |
| typescript | 4.5.5 | 4.5.5 | 4.7.4 | @openmsupply-client/electron | devDependencies | https://www.typescriptlang.org/ |  |
| webpack | 5.72.1 | 5.73.0 | 5.73.0 | openmsupply-client | devDependencies | https://github.com/webpack/webpack |  |
| webpack | 5.72.0 | 5.73.0 | 5.73.0 | @openmsupply-client/host | devDependencies | https://github.com/webpack/webpack |  |
| webpack-cli | 4.9.2 | 4.10.0 | 4.10.0 | @openmsupply-client/host | devDependencies | https://github.com/webpack/webpack-cli/tree/master/packages/webpack-cli |  |

# Patch
| Package | Current | Wanted | Latest | Package Type | URL | CHANGELOG |
|---|---|---|---|---|---|---|
| @babel/core | 7.18.2 | 7.18.6 | 7.18.6 | openmsupply-client | devDependencies | https://babel.dev/docs/en/next/babel-core |  |
| @babel/preset-env | 7.18.2 | 7.18.6 | 7.18.6 | openmsupply-client | devDependencies | https://babel.dev/docs/en/next/babel-preset-env |  |
| @capacitor/keyboard | 1.2.2 | 1.2.3 | 1.2.3 | openmsupply-client | devDependencies | https://github.com/ionic-team/capacitor-plugins#readme |  |
| @emotion/react | 11.9.0 | 11.9.3 | 11.9.3 | @openmsupply-client/common | dependencies | https://github.com/emotion-js/emotion/tree/main/packages/react |  |
| @fontsource/inter | 4.5.10 | 4.5.11 | 4.5.11 | @openmsupply-client/host | dependencies | https://github.com/fontsource/fontsource/tree/main/fonts/google/inter#readme |  |
| @graphql-codegen/near-operation-file-preset | 2.2.12 | 2.2.15 | 2.2.15 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @graphql-codegen/typescript-graphql-request | 4.4.8 | 4.4.11 | 4.4.11 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @graphql-codegen/typescript-msw | 1.0.10 | 1.0.13 | 1.0.13 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @graphql-codegen/typescript-operations | 2.4.0 | 2.4.3 | 2.4.3 | openmsupply-client | devDependencies | https://github.com/dotansimha/graphql-code-generator#readme |  |
| @mui/material | 5.8.1 | 5.8.7 | 5.8.7 | @openmsupply-client/common | dependencies | https://mui.com/material-ui/getting-started/overview/ |  |
| @storybook/addon-actions | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/addons/actions |  |
| @storybook/addon-essentials | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/addons/essentials |  |
| @storybook/addon-interactions | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/addons/interactions |  |
| @storybook/addon-links | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/addons/links |  |
| @storybook/addons | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/lib/addons |  |
| @storybook/builder-webpack5 | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/lib/core |  |
| @storybook/manager-webpack5 | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/lib/core |  |
| @storybook/react | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/app/react |  |
| @storybook/theming | 6.5.8 | 6.5.9 | 6.5.9 | openmsupply-client | devDependencies | https://github.com/storybookjs/storybook/tree/main/lib/theming |  |
| @swc/core | 1.2.194 | 1.2.210 | 1.2.210 | openmsupply-client | devDependencies | https://swc.rs |  |
| @testing-library/user-event | 14.2.0 | 14.2.1 | 14.2.1 | openmsupply-client | devDependencies | https://github.com/testing-library/user-event#readme |  |
| @types/react | 18.0.9 | 18.0.15 | 18.0.15 | openmsupply-client | devDependencies | https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react |  |
| @types/react-dom | 18.0.5 | 18.0.6 | 18.0.6 | openmsupply-client | dependencies | https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-dom |  |
| @vercel/webpack-asset-relocator-loader | 1.7.0 | 1.7.0 | 1.7.2 | @openmsupply-client/electron | devDependencies | https://github.com/vercel/webpack-asset-relocator-loader#readme |  |
| babel-jest | 28.1.0 | 28.1.2 | 28.1.2 | openmsupply-client | devDependencies | https://github.com/facebook/jest#readme |  |
| eslint-plugin-react | 7.30.0 | 7.30.1 | 7.30.1 | openmsupply-client | devDependencies | https://github.com/jsx-eslint/eslint-plugin-react |  |
| i18next | 21.8.4 | 21.8.13 | 21.8.13 | @openmsupply-client/common | dependencies | https://www.i18next.com |  |
| jest | 28.1.0 | 28.1.2 | 28.1.2 | openmsupply-client | devDependencies | https://jestjs.io/ |  |
| jest-environment-jsdom | 28.1.0 | 28.1.2 | 28.1.2 | openmsupply-client | devDependencies | https://github.com/facebook/jest#readme |  |
| react-query | 3.39.0 | 3.39.1 | 3.39.1 | @openmsupply-client/common | dependencies | https://github.com/tannerlinsley/react-query#readme |  |
| recharts | 2.1.10 | 2.1.12 | 2.1.12 | @openmsupply-client/common | dependencies | https://github.com/recharts/recharts |  |
| recharts | 2.1.10 | 2.1.12 | 2.1.12 | @openmsupply-client/dashboard | dependencies | https://github.com/recharts/recharts |  |
| serve | 13.0.2 | 13.0.4 | 13.0.4 | @openmsupply-client/host | devDependencies | https://github.com/vercel/serve#readme |  |
| ts-jest | 28.0.3 | 28.0.5 | 28.0.5 | openmsupply-client | devDependencies | https://kulshekhar.github.io/ts-jest |  |
| ts-loader | 9.3.0 | 9.3.1 | 9.3.1 | @openmsupply-client/electron | devDependencies | https://github.com/TypeStrong/ts-loader |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/common | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/config | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/dashboard | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/host | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/inventory | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/invoices | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/requisitions | devDependencies | https://www.typescriptlang.org/ |  |
| typescript | 4.7.2 | 4.7.4 | 4.7.4 | @openmsupply-client/system | devDependencies | https://www.typescriptlang.org/ |  |
| webpack-dev-server | 4.9.0 | 4.9.3 | 4.9.3 | @openmsupply-client/host | devDependencies | https://github.com/webpack/webpack-dev-server#readme |  |

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #309 

Updated all but the beta and rc versions.
Bonus! fixed the tests which my recent PR broke.

### CODE REVIEW

**Review by CarlosNZ — APPROVED** [review]
Cool, loaded and tested -- all seems good 👍

File: client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx [hunk 1]
```diff
@@ -6,69 +6,69 @@ import { TestingProvider } from '../../../../../utils';
...
+  //           onChange={jest.fn()}
+  //         />
+  //       </TableProvider>
+  //     </TestingProvider>
+  //   );
+
+  //   const node = getByRole('button', { name: 'page 1' });
+
+  //   expect(node).toBeInTheDocument();
+  // });
```
Comment (clemens-msupply) [inline client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx:67]: plan to move it back in? or remove entirely?

**Review by clemens-msupply — APPROVED** [review]
Thanks

For the tests I get:
```
console.error
      Warning: The current testing environment is not configured to support act(...)
```
is that an error on my side?

File: client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx (hunk 1 above)
Comment (mark-prins) [reply client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx:67]: oops. that was while fixing the pagination test. good spotting!

Comment (mark-prins) [reply client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx:67]: and for the tests.. no. When I upgraded the first time I tried this https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#configuring-your-testing-environment but it didn't help.

Comment (mark-prins) [reply client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx:67]: no, wait. good catch! In removing the warning, I've found a couple of tests that were giving a false positive. 
Have fixed up and the suite is looking sweet!

**Pull request merged**